dom/events/enter/enter-test.js

```diff
@@ -14,6 +14,9 @@ function makeKeyboardEvent() {
 		event = new KeyboardEvent("keyup", {
 			key: "Enter"
 		});
+		Object.defineProperty(event, "type", {
```

Comment:
I thought this is what John and I fixed.  I made sure that jQuery was passed a copied object.